global_planner/src/orientation_filter.cpp

```diff
@@ -109,6 +131,12 @@ void OrientationFilter::processPath(const geometry_msgs::PoseStamped& start,
                 set_angle(&path[i], angles::normalize_angle(tf2::getYaw(path[i].pose.orientation) + M_PI_2));
             }
             break;
+        case OMNI:
+            path[0].pose.orientation = start.pose.orientation;
+            for(int i=1;i<n-1;i++){
+                set_angle(&path[i], tf2::getYaw(path[i-1].pose.orientation));
+            }
```

Comment:
S: how about
```suggestion
            for(int i=0;i<n-1;i++){
                path[i].pose.orientation = start.pose.orientation;
            }
```
directly?